docs/Reminders.md

```diff
@@ -0,0 +1,10 @@
+??? ListItemButton vs ListItemLink vs MenuItemLink... maybe its just the naming that needs fixing
+??? When common folder vs component folder?
```

Comment:
Components are reusable components that are intended to be called `<Component />` while `common` contains functions / data structures that are called from other files. Hooks go into `common`.